.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15'
+      - name: Install hc-codesign
+        id: codesign
+        run: |
+          docker login docker.pkg.github.com -u docker -p $GITHUB_TOKEN && \
+          docker pull docker.pkg.github.com/hashicorp/hc-codesign/hc-codesign:$VERSION && \
+          echo "::set-output name=image::docker.pkg.github.com/hashicorp/hc-codesign/hc-codesign:$VERSION"
+        env:
+          VERSION: v0
+          GITHUB_TOKEN: ${{ secrets.CODESIGN_GITHUB_TOKEN }}
+      - name: Install wget & clamAV antivirus scanner
+        run : |
+           sudo apt-get -qq install -y ca-certificates wget clamav
+           wget --version
+      - name: Install maldet malware scanner
```

Comment:
I would recommend running the full install script, because it includes [a self-test that checks both scanners are actually working](https://github.com/hashicorp/packagespec/blob/master/assets/private/release/scripts/av-install#L68-L95). I think this is important because the scanner pulls down fresh definitions when it is run, so its behaviour is not guaranteed to be consistent. For packagespec I copied this whole script https://github.com/samsalisbury/av-scan/blob/master/install to `assets/private/release/scripts/av-install` and just call that from CI.

Reply:
Thanks Sam, I wasn't sure why these checks were here. If maldet or clamav aren't functioning properly, will the calls to them error out, or are there other 'hidden' failures that this will capture?

Reply:
The idea of the checks is to see whether we can trust the exit code from both tools. We first feed it a test blob that contains a known string that should be identified as malware and ensure that the exit code is not zero. Then feed it a benign blob and check that the exit code is zero.